source/_components/climate.intesishome.markdown

```diff
@@ -0,0 +1,58 @@
+---
+layout: page
+title: "IntesisHome AC Controller"
+description: "Instructions how to integrate IntesisHome AC devices with Home Assistant"
+date: 2019-03-12 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: intesishome.png
+ha_category: Climate
+ha_release: 
+ha_iot_class: "Cloud Push"
+---
+
+The `IntesisHome` climate platform let you control [IntesisHome](https://www.intesishome.com) devices. IntesisHome provide integrations with air conditioners including including Panasonic, Daikin, Fujitsu, Toshiba, LG and more.
+
+To set it up, add the following information to your `configuration.yaml` file:
+
+```yaml
+climate:
+  - platform: intesishome
+    username: YOUR_USERNAME
+    password: !secret intesishome_password
+```
+
+The password must be stored in your `secrets.yaml` file:
+```yaml
+intesishome_password: YOUR_PASSWORD
```

Comment:
I think we can skip this, put your password in the secrets.yam filel is not required, but something users can do optional.

Reply:
Thanks, have updated that now

Reply:
I must say that I started a discussion about it on Discord, because it would not be wrong to put the passwords in the `secrets.yaml` file and in the way you had previously drafted. Only at the moment it is not the "standard" that we use, who knows this may change in the future 😉